modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -205,13 +205,17 @@ TEST_P(Test_ONNX_layers, Convolution_variable_weight_bias)
 
 TEST_P(Test_ONNX_layers, Gather)
 {
-    if (backend == DNN_BACKEND_INFERENCE_ENGINE_NN_BUILDER_2019 && target == DNN_TARGET_MYRIAD)
-        applyTestTag(CV_TEST_TAG_DNN_SKIP_IE_MYRIAD, CV_TEST_TAG_DNN_SKIP_IE_NN_BUILDER);
+    if (backend != DNN_BACKEND_OPENCV || target != DNN_TARGET_CPU)
+        throw SkipTestException("Only CPU is supported");  // FIXIT use tags
```

Comment:
When do we lost OpenCV/OpenCL support?

Reply:
See the comment above.

Reply:
OpenCV/OpenCL code path should have automatic fallback implementation on CPU code.
Test should check that fallback works.

Reply:
Updated the PR.

Reply:
> Test should check that fallback works.

and OpenCL FP16 tests failed...